backend/test/modules/doubles-tournament/presentation/rest-api/DoublesTournamentRestApiModule.spec.ts

```diff
@@ -128,4 +130,205 @@ describe('Doubles Tournament REST API', () => {
+  });
+
+  //TODO check tests below
+  // it('POST /rest-api/doubles-tournaments/:tournamentId/start | tournament with such id is not ready to start', async () => {
+  //   //Given
+  //   const tournamentId = 'sampleTournament1Id';
+  //   const tournamentId2 = 'sampleTournament2Id';
+  //   const queryPublisher = QueryPublisherMock([
+  //     new DoublesTournament({
+  //       tournamentId: tournamentId,
+  //       tournamentTeams: [
+  //         new TournamentTeam({
+  //           teamId: TeamId.from('sampleTeamId1'),
+  //           firstTeamPlayer: 'samplePlayer1',
+  //           secondTeamPlayer: 'samplePlayer2',
+  //         }),
+  //         new TournamentTeam({
+  //           teamId: TeamId.from('sampleTeamId2'),
+  //           firstTeamPlayer: 'samplePlayer3',
+  //           secondTeamPlayer: 'samplePlayer4',
+  //         }),
+  //       ],
+  //     }),
+  //   ]);
+  //   const { agent } = testModuleRestApi(DoublesTournamentRestApiModule, { queryPublisher });
+  //
+  //   //When
+  //   const { body, status } = await agent.post(`/rest-api/doubles-tournaments/${tournamentId2}/start`).send({ tournamentId: tournamentId2 });
+  //
+  //   //Then
+  //   expect(queryPublisher.executeCalls).toBeCalledWith(new StartTournament({ tournamentId: tournamentId2 }));
+  //   expect(status).toBe(StatusCodes.NOT_FOUND);
+  //   expect(body).toStrictEqual({ message: "There aren't any tournaments ready to start dupa" });
+  // });
+  //
+  // it('POST /rest-api/doubles-tournaments/:tournamentId/start | when try to start twice the same tournament, then get error', async () => {
+  //   //Given
+  //   const tournamentId = 'sampleTournament1Id';
+  //   const queryPublisher = QueryPublisherMock([
+  //     new DoublesTournament({
+  //       tournamentId: tournamentId,
+  //       tournamentTeams: [
+  //         new TournamentTeam({
+  //           teamId: TeamId.from('sampleTeamId1'),
+  //           firstTeamPlayer: 'samplePlayer1',
+  //           secondTeamPlayer: 'samplePlayer2',
+  //         }),
+  //         new TournamentTeam({
+  //           teamId: TeamId.from('sampleTeamId2'),
+  //           firstTeamPlayer: 'samplePlayer3',
+  //           secondTeamPlayer: 'samplePlayer4',
+  //         }),
+  //       ],
+  //     }),
+  //   ]);
+  //   const { agent } = testModuleRestApi(DoublesTournamentRestApiModule, { queryPublisher });
+  //
+  //   //When
+  //   await agent.post(`/rest-api/doubles-tournaments/${tournamentId}/start`).send({ tournamentId: tournamentId });
+  //
+  //   const { body, status } = await agent
+  //     .post(`/rest-api/doubles-tournaments/${tournamentId}/start`)
+  //     .send({ tournamentId: tournamentId });
+  //
+  //   //Then
+  //   expect(queryPublisher.executeCalls).toBeCalledWith(new StartTournament({ tournamentId: tournamentId }));
+  //   expect(status).toBe(StatusCodes.BAD_REQUEST);
+  //   expect(body).toStrictEqual({ message: 'Such tournament was already started' });
+  // });
```

Comment:
@nowakprojects mowa o tych testach zakomentowanych powyżej :-)

Reply:
Jak poprawisz kwestie architektury to pewnie wszystko Ci sie wyklaruje :) Pełno niepotrzebnej roboty z tymi testami tutaj. Na warstwie application testujemy zachowanie ,a prezentacje testujemy tylko czy odpowiednio deleguje prace do application.

Reply:
tak tak, nie wiem czemu powrzucałem tam do posta te if'y